DigitalLearningSolutions.Web/Helpers/ExternalApis/MapsApiHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace DigitalLearningSolutions.Web.Helpers.ExternalApis
+{
+    using System.Net.Http;
+    using System.Threading.Tasks;
+    using DigitalLearningSolutions.Data.Models.External.Maps;
+    using Newtonsoft.Json;
+
+    public interface IMapsApiHelper
+    {
+        Task<MapsResponse> GetPostcodeCoordinates(string postcode);
+    }
+
+    public class MapsApiHelper : IMapsApiHelper
+    {
+        private static readonly HttpClient Client = new HttpClient();

```

Comment:
Is this (or the MapsApiHelper) scoped properly? I know HttpClient is something you have to be careful with. We don't want to be creating multiple instances of HttpClient (or multiple instances of MapsApiHelper containing multiple)

Reply:
From what I've read up, I suspect that it's fine. But just to be safe I've changed it about to make it definitely rely on an injected httpClient.

Reply:
Yeah, I thought it might be but wasn't sure. Nice change to use one of the patterns recommended by the asp net core docs though.